service/controller/clusterapi/machine_deployment.go

```diff
@@ -31,17 +31,31 @@ type MachineDeploymentConfig struct {
 	Locker       locker.Interface
 	Logger       micrologger.Logger
 
+	CalicoCIDR                 int
```

Comment:
Wiring all the information necessary for Cloud Config rendering since we render for each machine deployment now.